src/custom/state/orders/priceUtils.ts

```diff
@@ -53,3 +55,13 @@ export function calculatePrice(params: CalculatePriceParams): BigNumber {
 export function invertPrice(price: BigNumber): BigNumber {
   return ONE_BIG_NUMBER.div(price)
 }
+
+export function fractionToPrice(fraction: Fraction, inputCurrency: Token, outputCurrency: Token): Price<Token, Token> {
+  // Note that here the fraction shows the price in units (for both tokens). The Price class is decimals aware, so we need to adapt it
+  const adjustedFraction = adjustDecimalsAtoms(fraction, inputCurrency.decimals, outputCurrency.decimals)
+
+  return new Price({
+    quoteAmount: CurrencyAmount.fromRawAmount(outputCurrency, adjustedFraction.numerator),
+    baseAmount: CurrencyAmount.fromRawAmount(inputCurrency, adjustedFraction.denominator),
+  })
+}
```

Comment:
Maybe this belong to https://github.com/cowprotocol/cowswap/blob/c2ed0796ff810ce060a7656021f4f3a9c1a6cbe3/src/cow-react/utils/fractionUtils.ts#L7 

Or maybe not, it's price related...